packages/documentation/pages/examples/layouts.vue

```diff
@@ -22,7 +23,22 @@
 			headerTitle="LayoutContainer Example"
 			:menu="actionbarMenu"
 		/>
-		<div slot="workspace"></div>
+		<div slot="workspace">
+			<KtForm>
+				<KtFieldSingleSelect
+					v-model="theme"
+					formKey="NONE"
+					isOptional
+					label="Navbar Theme"
+					:options="[
+						{ label: 'Reversed theme', value: 'reverse' },
+						{ label: 'Light theme', value: 'light' },
+						{ label: 'Dark theme', value: 'dark' },
+						{ label: 'None', value: null },
```

Comment:
also if you want a  null option explicity, pass `hideClear` so u don't have a null option AND the option to clear which will also have a vaule null but a label `''`

Reply:
I can't name it "None" then right?? I think empty label is a bit confusing?

Reply:
No you can keep the name, I'm suggesting you add the `hideClear` flag on the component because you have a null option but also you can clear so you can two possible ways of having a null value and i don't think you need that given that you have explicitly a null option